components/eamxx/docs/technical/index.md

```diff
@@ -1,3 +1,24 @@
-# SCREAM Technical Guide
+# EAMxx Technical Guide
 
-SCREAM contributors and maintainers will add detailed technical information about SCREAM here.
+The goal of this document is to describe the specific equations, parameterizations, and numerical methods used in the current version of EAMxx. Because our master-branch implementation changes every time we make a new commit, this documentation will also evolve continuously. As such, documentation for master should always be considered to be preliminary and under construction. If you want trustworthy documentation, pull it from an official model release. 
+
+## Overview
+
+Currently, EAMxx is only configured for km-scale convection-permitting runs. In order to provide scientifically-credible simulations at lower resolution, parameterizations for the following processes would be needed:
+
+1. deep convection
+2. gravity-wave drag
+3. energy fixer
+
+The only configuration of EAMxx that is currently implemented is the convection-permitting version, commonly known as the Simple Cloud-Resolving E3SM Atmosphere Model (SCREAM). Processes in EAMxx-SCREAM are:
+
+1. a non-hydrostatic version of the **spectral-element dynamical core** used by other E3SM Atmosphere Model versions [@Taylor_et20] with semi-Lagrangian tracer advection as described by [@Bradley_et22]
```

Comment:
Here and other places: 
For citations, we still need to write out the citation. The references in brackets only give the numbers. 
For example, in the [ELM TOP description](https://github.com/E3SM-Project/E3SM/blob/master/components/elm/docs/tech-guide/top_solar_parameterization.md), the citation is written out as: 
```
  Fig 1.: A schematic diagram of the five components of incoming solar radiation over mountains adapted from Lee et al. (2011)[@lee2011parameterization]. 
```